Improve "t" filter docs (#19173)
OrchardCMS/OrchardCore: src/docs/reference/modules/Liquid/README.md

Bonjour!
```

> There is an option to pass parameter(s) to the localized string:

Reviewer: I think this could be a little bit more explicit, e.g.:

```suggestion
You can pass one or more parameters to a localized string:
```

Input

```liquid
{{ "Hello {0}!" | t: "John" }}
```

Output

```text
Bonjour John!
```

> 

Reviewer: Maybe add an explanation:

```suggestion
In this example, the parameter "John" is injected into the placeholder `{0}` in the translated string.

```

Author: Agree, but if it's in the localization module, otherwise this page will be crowded :)

So what do you think?

Reviewer: I thought just below the new example, nothing fancy.

Reviewer: IMHO it's also clear as-is. Whatever you prefer...

Author: Let's go with this, but I will consider the change if we move the filters to each related module coz we need such details
